net/freeradius-server/Makefile

```diff
@@ -0,0 +1,605 @@
+#
+# Copyright (C) 2008-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=freeradius3
+PKG_VERSION:=release_3_0_11
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/FreeRADIUS/freeradius3/archive
```

Comment:
your sed was a bit agressive, url is still "https://github.com/FreeRADIUS/freeradius-server/archive"